x-pack/filebeat/module/crowdstrike/falcon/test/falcon-events.log-expected.json

```diff
@@ -0,0 +1,572 @@
+        "service.type": "crowdstrike",
+        "input.type": "log",
+        "event.module": "crowdstrike",
+        "event.dataset": "crowdstrike.falcon",
+        "event.kind": "event",
+        "event.action": "remote_response_session_end_event",
+        "event.type": "info",
+        "host.name": "hostnameofmachine",
+        "log.file.path": "falcon-events.log",
+        "log.flags": [
+            "multiline"
+        ],
+        "log.offset": 457,
+        "agent.type": "falcon"
+    },
+    {
+        "@timestamp": "2020-02-19T08:30:00.000Z",
+        "process.pid": 38684386611,
+        "process.name": "explorer.exe",
+        "process.executable": "C:\\Windows\\Explorer.EXE",
+        "event.dataset": "crowdstrike.falcon",
+        "event.kind": "alert",
+        "event.action": "malware_ransomware",
+        "event.type": "info",
+        "event.category": "malware",
+        "event.severity": 4,
+        "event.module": "crowdstrike",
+        "service.type": "crowdstrike",
+        "user.name": "alice",
+        "user.domain": "CORP-DOMAIN",
+        "rule.reference": "https://falcon.crowdstrike.com/ec86abd353824e96765ecbe18eb4f0b4",
+        "log.flags": [
+            "multiline"
+        ],
+        "log.offset": 911,
+        "log.file.path": "falcon-events.log",
+        "source.ip": "192.168.12.51",
+        "agent.type": "falcon",
+        "host.name": "alice-laptop",
+        "message": "Terminated a process related to the deletion of backups, which is often indicative of ransomware activity.",
+        "fileset.name": "falcon",
+        "input.type": "log",
+        "crowdstrike.metadata.customerIDString": "8f69fe9e-b995-4204-95ad-44f9bcf75b6b",
+        "crowdstrike.metadata.offset": 294564,
+        "crowdstrike.metadata.eventType": "DetectionSummaryEvent",
+        "crowdstrike.metadata.eventCreationTime": 1582101000000,
+        "crowdstrike.metadata.version": "1.0",
+        "crowdstrike.event.ParentProcessId": 38682494050,
+        "crowdstrike.event.SHA256String": "6a671b92a69755de6fd063fcbe4ba926d83b49f78c42dbaeed8cdb6bbc57576a",
+        "crowdstrike.event.SensorId": "7c808b4c8878433287eea53d4a8c3268",
+        "crowdstrike.event.LocalIP": "192.168.12.51",
+        "crowdstrike.event.FalconHostLink": "https://falcon.crowdstrike.com/ec86abd353824e96765ecbe18eb4f0b4",
+        "crowdstrike.event.Tactic": "Malware",
+        "crowdstrike.event.ProcessEndTime": 0,
+        "crowdstrike.event.Severity": 4,
+        "crowdstrike.event.CommandLine": "C:\\Windows\\Explorer.EXE",
+        "crowdstrike.event.Technique": "Ransomware",
+        "crowdstrike.event.Objective": "Falcon Detection Method",
+        "crowdstrike.event.ProcessId": 38684386611,
+        "crowdstrike.event.DetectDescription": "Terminated a process related to the deletion of backups, which is often indicative of ransomware activity.",
+        "crowdstrike.event.PatternDispositionDescription": "Prevention, process killed.",
```

Comment:
This field sounds perfect to populate `event.action`. Unfortunately it sounds like it's not reliably populated. It's only on 1 of the 15 events

Reply:
I think it's okay to use it here for this event type. Made the change.